values.yaml

```diff
@@ -289,6 +289,14 @@ global:
     extraLabels: []
       # do-funky-injection: maybe
 
+  # Enable default mysql database.
+  # This is not recommended for production.
+  mysql:
+    enabled: false
+    servicePort: "3306"
+    username: "root"
+    password: "root"
```

Comment:
Currently this PR does not have any tests in tests/chart_tests. I think we should make some before merging the PR. This bit of code would be a good place to start.